packages/web/src/shared/components/form/AutocompleteInput.tsx

```diff
@@ -41,13 +41,14 @@ const AutocompleteInput = ({
   useEffect(() => {
     if (normalizedInputValue) {
       const filteredItems = items.filter((item) =>
-        item.label
-          .normalize()
-          .toLowerCase()
-          .includes(normalizedInputValue.toLowerCase())
+        ignoreDiacritics(item.label.normalize('NFD').toLowerCase()).includes(
+          ignoreDiacritics(normalizedInputValue.toLowerCase())
+        )
       );
       const noExactMatch = filteredItems.every(
-        (item) => item.label.normalize() !== normalizedInputValue
+        (item) =>
+          ignoreDiacritics(item.label.normalize('NFD')) !==
```

Comment:
Do you think the exact match should not ignore diacritics? I think this will make it difficult to add new glosses that have the same letters but different diacritics

Reply:
Yeah, I agree. I wasn't clear about how exactly you wanted the autocomplete matching to work.

Reply:
Try my newest commit.  

<img width="132" alt="image" src="https://github.com/arrocke/gloss-translation/assets/17209828/61dc8b00-9d9b-400a-ade2-99a42fa00660">

I think this behavior is good. It selects by default the other version, but still gives the user the option to create a new gloss if they want.